packages/protocol/thread/thread.ts

```diff
@@ -553,25 +475,19 @@ class _ThreadFront {
     return lineLocations;
   }
 
-  async setBreakpoint(initialSourceId: SourceId, line: number, column: number, condition?: string) {
+  async setBreakpoint(sourceId: SourceId, line: number, column: number, condition?: string) {
     try {
       this._invalidateResumeTargets(async () => {
         assert(this.sessionId, "no sessionId");
-        await this.ensureAllSources();
-        const sourceIds = this.getCorrespondingSourceIds(initialSourceId);
-        await Promise.all(
-          sourceIds.map(async sourceId => {
-            await ThreadFront.getBreakpointPositionsCompressed(sourceId);
-            const location = { sourceId, line, column };
-            const { breakpointId } = await client.Debugger.setBreakpoint(
-              { location, condition },
-              this.sessionId!
-            );
-            if (breakpointId) {
-              this.breakpoints.set(breakpointId, { location });
-            }
-          })
+        await ThreadFront.getBreakpointPositionsCompressed(sourceId);
```

Comment:
:hand: shouldn't it be `this.` instead of `ThreadFront.`?

Reply:
Agreed, although I didn't actually change that (and there are several other instances of this in `ThreadFront`)